### ISSUE TITLE
Arrow keys skipping accounts in list

### ISSUE THREAD

**Issue by awhalbert** [Authenticator-Extension/Authenticator#1083]
### Describe the issue

When I first open the extension, if I press "down", it takes me to the 3rd entry in the list. Every "down" stroke after that skips every other account.

The "right" arrow seems to work mostly correctly, but the "left" arrow also skips accounts. It's nearly impossible to select the 2nd entry in the list.

I've verified that my keyboard is not sending double signals. I tested these strokes other apps, and they work as expected.

### Browser

Chrome

### Browser Version

Version 114.0.5735.199 (Official Build) (64-bit)

### Extension Version

6.3.5

**Comment by Sneezry** [comment]
Does the up key work as expected?

**Comment by Sneezry** [comment]
Are you using Korean input? It seems there's a bug for Korean input when trigger down key event. https://github.com/vuejs/vue/issues/10277

**Comment by awhalbert** [comment]
No, the up key also skips accounts, and I am using the English keyboard layout.

**Comment by Sneezry** [comment]
Emmm... We use exactly the same logic code for both up/down and left/right keys to go through the codes

https://github.com/Authenticator-Extension/Authenticator/blob/c5976b63c5ff3b8c6a1e2ba5077071abf973aead/src/components/Popup/MainBody.vue#L29-L32

That could be some issue with the framework.

Can you also check if you can see double ArrowDown when you click down key in this test page: https://jsfiddle.net/posva/kyjfca39/

**Comment by awhalbert** [comment]
I tried out the jsfiddle and it registered single events:
![image](https://github.com/Authenticator-Extension/Authenticator/assets/7232559/0e5c5597-8d20-4a44-8bb0-b69dd883f59a)

**Comment by awhalbert** [comment]
P.S. I also tried in an Incognito window and I also tried disabling all other extensions and restarting Chrome, but got the same result. I'm using Windows 11 and was not able to reproduce on a Winodws 10 machine.

**Comment by awhalbert** [comment]
P.P.S. The extension works correctly in Firefox on the same machine.

I also tried creating a backup, uninstalling / reinstalling the Chrome extension, and re-importing my backup, which did not fix the issue in Chrome.

I also tried deleting all entries and creating some new ones manually, which also did not fix the issue in Chrome.

**Comment by Sneezry** [comment]
Thanks for the detailed information. I guess that's more like the framework issue. I need to investigate the root cause. However, because I cannot reproduce this issue on my side, it may take some time.
